Wrong assertion in creation of physical groups, 'pygmsh.built_in.ellipse_arc.EllipseArc'>.

With the later `pygmsh (5.0.1)`, it is no longer possible to add physical lines to EllipseArc
MWE:
```
from pygmsh.built_in.geometry import Geometry 
res = 0.0125 
c_x = 0.5 
c_y = 0.5 
r_x = 0.025 
geometry = Geometry() 
c = geometry.add_point((c_x,c_y,0))     
p1 = geometry.add_point((c_x-r_x, c_y,0),lcar=res) 
p2 = geometry.add_point((c_x, c_y+r_x,0),lcar=res) 
p3 = geometry.add_point((c_x+r_x, c_y,0),lcar=res) 
p4 = geometry.add_point((c_x, c_y-r_x,0),lcar=res) 
arc_1 = geometry.add_ellipse_arc(p1, c, p2, p2) 
arc_2 = geometry.add_ellipse_arc(p2, c, p3, p3) 
arc_3 = geometry.add_ellipse_arc(p3, c, p4, p4) 
arc_4 = geometry.add_ellipse_arc(p4, c, p1, p1) 
obstacle_loop = geometry.add_line_loop([arc_1, arc_2, arc_3, arc_4]) 
geometry.add_physical_line(obstacle_loop.lines, label=1)
```
yielding:
```
AssertionError: Can add physical groups only for Points, Lines, Surfaces, Volumes, not <class 'pygmsh.built_in.ellipse_arc.EllipseArc'>.
```

--- a/pygmsh/built_in/geometry.py
+++ b/pygmsh/built_in/geometry.py
@@ -184,8 +184,7 @@ class Geometry(object):
                 e,
                 (
                     Point,
-                    Line,
-                    CircleArc,
+                    LineBase,
                     Surface,
                     PlaneSurface,
                     SurfaceBase,

